### DISCUSSION TITLE
How should I enable going back to previous page with media element?

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [CommunityToolkit/Maui#941] [Q&A]
I have my app full screen while playing video using media element. I am looking at ways to go back to previous page when playing on Android. Currently the workaround is to swipe up tap on back button. I would like to use gesture recognizer and swipe right. That is blocked by exoplayer and I saw on closed bug report that it was that way deliberately. So I am currently swiping up and clicking on back button atm to go back. This works for developing app but I have made app full screen so I need some sort of control that pops up when i touch the screen. The playback controls block gestures, but allow tap. 

One option is to have the full screen controls show back up when I click pause. When I start playback again it could go back to full screen. But I have no idea how to implement that. I would be happy to do it this way. I think this is the way to go. I don't think it would be easy or simple to override exoplayer using a handler to modify the controls. It blocks touch for a reason. I may not know that reason but I imagine it is that way for a purpose.

I am hoping someone might have a better way to handle full screen mode and might have a suggestion for how to do the ui element so I can have controls and have a method of getting back to previous page that is intuitive for the user. I appreciate any suggestions on what I should do.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
